hackathon_site/dashboard/frontend/src/App.js

```diff
@@ -1,22 +1,34 @@
 import React from "react";
 import { HashRouter as Router, Route } from "react-router-dom";
 import { StylesProvider } from "@material-ui/core/styles";
-import NavBar from "components/general/Navbar/Navbar";
+import { Provider as ReduxProvider } from "react-redux";
+
+import store from "slices/store";
+
 import "App.scss";
+import NavBar from "components/general/Navbar/Navbar";
+import Greeting from "components/general/Greeting/Greeting";
 
-function App() {
+const UnconnectedApp = () => {
     return (
         <div className="App">
-            <StylesProvider injectFirst>
-                <Router>
-                    <NavBar />
-                </Router>
-                <div className="App-header">
-                    <p>IEEeeeeeee</p>
-                </div>
-            </StylesProvider>
+            <Router>
+                <NavBar />
+            </Router>
+            <div className="App-header">
+                <p>IEEeeeeeee</p>
+                <Greeting userID={1} />
+            </div>
         </div>
     );
-}
+};
+
+const ConnectedApp = () => (
+    <ReduxProvider store={store}>
+        <StylesProvider injectFirst>
+            <UnconnectedApp />
+        </StylesProvider>
+    </ReduxProvider>
+);
 
-export default App;
+export default ConnectedApp;
```

Comment:
what is App doing diff between connected and unconnected

Reply:
This is all about Redux - a major library that we're integrating into the front end.